deepmd/common.py

```diff
@@ -42,6 +42,12 @@
     "float64": tf.float64,
 }
 
+PRECISION_MAPPING: Dict[int, type] = {
+    1: np.float32,
+    2: np.float64,
+    19: np.float16,
+}
```

Comment:
It can be replaced by [`DType.as_numpy_dtype`](https://www.tensorflow.org/api_docs/python/tf/dtypes/DType#attributes).

Reply:
The node.dtype is 'int' type, which is something different with the 'tf.Dtype' type, so it could not be replaced by as_numpy_dtype directly.